### PR TITLE
Add structured data to docs

### PR DESCRIPTION
**Pull request by adrianspdev**
### Context
This PR includes ld+json script with additional SEO information

### How has this been tested?
Locally

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/dev-handsontable/issues/1821

### Affected project(s):
- [x] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:
- [x] Docs pages contains ld+json script

[skip changelog]

### CODE REVIEW

File: docs/.vuepress/theme/components/SchemaStructuredData.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      v-html="meta_structuredData"
+    ></script>
+  </div>
+</template>
+
+<script>
+import * as dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+const timezone = require('dayjs/plugin/timezone');
```
Comment (budnix) [inline docs/.vuepress/theme/components/SchemaStructuredData.vue:15]: Is there any chance of using `import` here? It looks a bit odd to see ESM and CommonJS mixed together 😄

**Review by budnix — APPROVED** [review]
All good 👍 I left one note to consider.

File: docs/.vuepress/theme/components/SchemaStructuredData.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        : `${this.$page.hostname}/docs${this.$page.path}`;
+    },
+
+    // Inject Schema.org structured data
+    meta_structuredData() {
+      const organization = this.$site.themeConfig.organization;
+      const structuredData = {
+        '@context': 'https://schema.org/',
+        '@type': 'WebPage',
+        name: this.meta_data.title || null,
```
Comment (evanSe) [inline docs/.vuepress/theme/components/SchemaStructuredData.vue:66]: Is it possible to have name, headline and description to be a bit more descriptive like the original issue example json ld

**Review by evanSe — COMMENTED** [review]
Could you cross reference with the original json ld example with the actual generated examples, some improvements could be made.

**Review by budnix — DISMISSED** [review]

**Review by evanSe — APPROVED** [review]

**Review by budnix — CHANGES_REQUESTED** [review]
I tried to push the changes on staging but it throws a lot of errors https://github.com/handsontable/handsontable/actions/runs/8721068534/job/23923981279?pr=10880. Can you check that?

**Review by budnix — APPROVED** [review]
I don't know if it's ok that the structure is generated by JS only. if so, it's ok for me.

**Review by budnix — APPROVED** [review]

**Pull request merged**